template-sample-app/template/src/components/views/Kpi.js

```diff
@@ -44,18 +43,27 @@ export default function Kpi() {
         transitionDuration: 500,
       })
     );
-
+    // Add the source query for the KPI
```

Comment:
I honestly would remove all comments now from this useEffect part. After the refactor, the code is clear enough (it wasn't previously). What do you think?

Reply:
Yep, I'm going to remove it.